MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "j@docker.com"
```

Comment:
i stole it hehehe

On Fri, May 22, 2015 at 10:12 AM, Michael Crosby <notifications@github.com>
wrote:

> In MAINTAINERS
> <https://github.com/docker/docker/pull/13391#discussion_r30913046>:
>
> > @@ -591,7 +598,7 @@ made through a pull request.
> >
> >  	[people.jfrazelle]
> >  	Name = "Jessie Frazelle"
> > -	Email = "jess@docker.com"
> > +	Email = "j@docker.com"
>
> what is this!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13391/files#r30913046>.
>